translate/rus.json

```diff
@@ -157,7 +157,7 @@
 "fifty-move rule":"правило 50 ходов",
 "final decision":"окончательное решение",
 "Final FEN":"окончательное FEN",
-"final":"финал",
+"Final":"финал",
```

Comment:
Lines 160 and 267: is Caps that important or not? Not to confuse with line 367.

Reply:
The GUI understands the cases, so if the json contains "final":"финал", then when the GUI wants to display "final", it will display "финал", and when it wants to display "Final", it will display "Финал", etc.
If you make that change above, then the first case will not work anymore, because "final" won't be found, and in the second case, Final will be translated to финал instead of Финал.

Reply:
Ok, so consider this change fully reverted back, do not apply it.